releasenotes/notes/botocore_dsm_sqs-02df9afa4b4bb503.yaml

```
@@ -0,0 +1,3 @@
+---
+features:
+  - Added message attributes to support data streams monitoring
```

Feel free to update the text. I just wanted to fix the formatting and tensing. Here are the guidelines to writing a release note: https://github.com/DataDog/dd-trace-py/blob/1.x/docs/releasenotes.rst#release-note-sections.

```suggestion
  borocore: Extends data streams monitoring support to sqs botocore.
```